configuration/scripts/machines/env.badger_intel

```diff
@@ -12,6 +12,14 @@ if ("$inp" != "-nomodules") then
 #module purge
 #module load intel
 #module load openmpi
+module unload hdf5-serial
+module unload hdf5-parallel
+module unload netcdf-serial
+module unload netcdf-h5parallel
+module load hdf5-serial
```

Comment:
I am curious why we unload and load the same module

Reply:
I've never paid attention to these files for different tested machines since we don't use them. I don't know why the load/unload is happening.